### ISSUE TITLE
RDP support

### ISSUE THREAD

**Issue by nbulusanjr** [jimradford/superputty#685]

**Comment by larsen0815** [comment]
There are already tools for this, e.g. mRemoteNG.

**Comment by bluikko** [comment]
@larsen0815 yes there is mRemoteNG but it is unbelievably buggy. For example there is no support for switching tags with ctrl-tab (or whatever).

+1 for this feature request
